build.sh

```diff
@@ -793,17 +793,29 @@ build_native $__SkipCoreCLR "$__BuildArch" "$__IntermediatesDir" "$__ExtraCmakeA
 # Build cross-architecture components
 if [ $__CrossBuild == 1 ]; then
     __SkipCrossArchBuild=1
+    TARGET_ROOTFS=""
+    export CROSSCOMPILE=0
     if [ $__DoCrossArchBuild == 1 ]; then
         # build cross-architecture components for x86-host/arm-target
         if [[ "$__BuildArch" == "arm" && "$__CrossArch" == "x86" ]]; then
             __SkipCrossArchBuild=0
+            if [[ "$__HostArch" == x64 ]]; then
```

Comment:
@janvorli Is it need to consider cross-architecture component in host-arm64/target-arm?